nfd/node-feature-discovery/templates/topologyupdater.yaml

```diff
@@ -0,0 +1,156 @@
+{{- if .Values.topologyUpdater.enable -}}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "node-feature-discovery.fullname" . }}-topology-updater
```

Comment:
General question for all the names in the templates:  currently ( before this PR) all the names are hardcoded. Are we expecting to support more then one NFD CR per cluster?